.github/workflows/npm-update-pf.yml

```diff
@@ -28,8 +28,17 @@ jobs:
           mkdir -p ~/.config/cockpit-dev
           echo ${{ github.token }} >> ~/.config/cockpit-dev/github-token
           eval $(ssh-agent)
-          ssh-add - <<< '${{ secrets.DEPLOY_KEY }}'
+          ssh-add - <<< '${{ secrets.NODE_CACHE_DEPLOY_KEY }}'
           bots/npm-update @patternfly >&2
           ssh-add -D
           ssh-agent -k
 
+      - name: Force push the change to trigger testing workflows
+        run: |
+          sleep 1 # make sure the timestamp changes
```

Comment:
Which timestamp? Of the git commit? I.e. to ensure a new SHA?

Reply:
Exactly.